localisation/00_zuwc_united_localisation.csv

```diff
@@ -3041,22 +3041,23 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
 nick_the_thunderous;the Thunderous;;;;;;;;;;;;;x
 nick_the_lightning_fist;the Lightning-Fist;;;;;;;;;;;;;x
 DEATH_WANDERING_ISLE_NO_KILLER;went to the Wandering Isle;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+ANGER_OPTION;Get out!;;;;;;;;;;;;;x
+DESPAIR_OPTION;We're doomed!;;;;;;;;;;;;;x
+DOUBT_OPTION;Traitors are everywhere!;;;;;;;;;;;;;x
+FEAR_OPTION;They're outside!;;;;;;;;;;;;;x
+HATRED_OPTION;I hate it!;;;;;;;;;;;;;x
+PRIDE_OPTION;I'm the heart of Pandaria!;;;;;;;;;;;;;x
+VIOLENCE_OPTION;Violence is the answer!;;;;;;;;;;;;;x
+infested_by_sha_of_anger;Infested by Sha of Anger;;;;;;;;;;;;;x
+infested_by_sha_of_despair;Infested by Sha of Despair;;;;;;;;;;;;;x
+infested_by_sha_of_doubt;Infested by Sha of Doubt;;;;;;;;;;;;;x
+infested_by_sha_of_fear;Infested by Sha of Fear;;;;;;;;;;;;;x
+infested_by_sha_of_hatred;Infested by Sha of Hatred;;;;;;;;;;;;;x
+infested_by_sha_of_pride;Infested by Sha of Pride;;;;;;;;;;;;;x
+infested_by_sha_of_violence;Infested by Sha of Violence;;;;;;;;;;;;;x
+EVTDESC_WCSHA_110;For the last couple of days, you have been feeling someone's presence. With peripheral vision, you notice some black slime in the long corridors of your house. Your mystics call them evil spirits or sha. Legends tell that they are all that have left of Y'Shaarj, an Old God. It looks like you attract them.;;;;;;;;;;;;;x
+EVTDESC_WCSHA_120;My thoughts unravel... like frayed rope... it looked like someone has possessed you. You don't see the black slime anymore and feel relieved. The sha vanished as suddenly as they had appeared.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCSHA_120;Your thoughts unravel... like frayed rope... it looked like someone has possessed you. You don't see the black slime anymore and feel relieved. The sha vanished as suddenly as they had appeared.;;;;;;;;;;;;;x
```

Reply:
Done in f2a78ff4edab746a8b1bf46305d127cd04c46929.